adapter/exporter/src/main/java/org/mobilitydata/gtfsvalidator/exporter/ProtobufNoticeExporter.java

```diff
@@ -63,7 +65,7 @@ public void export(final NonStandardHeaderNotice toExport) throws IOException {
                 .setCsvFileName(toExport.getFilename())
                 .setType(GtfsValidationOutputProto.GtfsProblem.Type.TYPE_UNKNOWN_COLUMN)
                 .setSeverity(GtfsValidationOutputProto.GtfsProblem.Severity.SUSPICIOUS_WARNING)
-                .setAltEntityValue(toExport.getExtraHeader())
+                .setAltEntityValue((String) toExport.getNoticeSpecific(NOTICE_SPECIFIC_KEY__EXTRA_HEADER_NAME))
```

Comment:
Looks like there is an extra `_` in `NOTICE_SPECIFIC_KEY__EXTRA_HEADER_NAME`? Or was that intentional to separate the key name?

Reply:
Looking at other entries below, I'm guessing it's intentional. Maybe it would be better to just remove most of the prefix, so you're left with something like `KEY_EXTRA_HEADER_NAME`?

Reply:
Streamlining indeed, will do.